gallery/src/components/NxDropdown/NxDropdownNavigationExample.tsx

```diff
@@ -14,10 +14,7 @@ function NxDropdownNavigationExample() {
       onClick = () => { alert('click'); };
 
   return (
-    <NxDropdown label="Navigation"
-                className="nx-dropdown--navigation"
```

Comment:
This class wasn't doing anything anymore (the CSS props it added were already there) so I removed it.